lib/volta.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+import os
+import platform
+from shutil import which
+
+from devenv.constants import root
+from devenv.lib import archive
+from devenv.lib import fs
+from devenv.lib import proc
+
+VOLTA_HOME = f"{root}/volta"
+
+_version = "1.1.1"
```

Comment:
Why such an ancient version?
The current version is 1.18.1:
https://github.com/Homebrew/homebrew-cask/blob/f398e64fd/Casks/v/volta.rb#L4

Their history doesn't go as far back as 1.1.1, but 1.15.0 is four months old:
https://github.com/Homebrew/homebrew-cask/blame/350bb474/Casks/volta.rb#L4C6-L4C19

Reply:
oh cool, i was just going with whatever sentry-bootstrap already had, will upgrade

Reply:
No idea what that 1.18.1 is, 1.1.1 is the latest upstream: https://github.com/volta-cli/volta